zeth_contracts/contracts/MerkleTreeMiMC7_test.sol

```diff
@@ -4,7 +4,8 @@
 
 pragma solidity ^0.5.0;
 
-import "./MerkleTreeMiMC7.sol";
+import "./BaseMerkleTree.sol";
```

Comment:
This is already fixed on #333

Reply:
I've rebased onto the target branch.  thanks.